src/main/java/com/thirdcc/webapp/service/impl/EventAttendeeServiceImpl.java

```diff
@@ -61,6 +103,12 @@ public EventAttendeeDTO save(EventAttendeeDTO eventAttendeeDTO) {
             .map(eventAttendeeMapper::toDto);
     }
 
+    @Override
+    @Transactional(readOnly = true)
+    public Page<EventAttendeeDTO> findAllByEventId(Pageable pageable, Long eventId) {
+        return eventAttendeeRepository.findAllByEventId(pageable, eventId)
```

Comment:
what if the eventId is not existed in the database?
like there are only 2 event in database with id 1 and id 2 but the API is calling for attendee with eventId 5